src/app/app.js

```diff
@@ -1,5 +1,5 @@
-// import Store from './store.local.js'
-import Store from './store.remote.await.js'
+import Store from './store.local.js'
```

Comment:
mit diesem PR hast du den localstorage aktiviert. gemäss titel des PR wolltest du aber den remote store aktieren?

Reply:
mir war nicht bewusst, dass der erste commit "changing to local store" auch gepusht wird. Ich hab aus versehen ein neuer branch aus dem vorgängigen branch gemacht.